### PR TITLE
New package Akko.CloudDriver

### PR DESCRIPTION
**Pull request by FaserF**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/172066)

### CODE REVIEW

**Comment by FaserF** [comment]
Does the validation-domain come because the file is hosted at:  https://files.monsgeek.com ?
This is the official download for the program, see: https://en.akkogear.com/download/ - despite the fact, that a slightly newer version has been published.

File: manifests/a/Akko/CloudDriver/370.1.32/Akko.CloudDriver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=QUSU.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# Latest download URL: https://www.mindmanager.com/mm-win-enterprise-admin-64
+
+PackageIdentifier: Akko.CloudDriver
+PackageVersion: 370.1.32
+MinimumOSVersion: 10.0.0.0
+Scope: user
+UpgradeBehavior: install
+Installers:
```
Comment (stephengillie) [inline manifests/a/Akko/CloudDriver/370.1.32/Akko.CloudDriver.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @FaserF,

It appears that `VCRedist` is a dependency:
image

**Comment by stephengillie** [comment]
> Does the validation-domain come because the file is hosted at: https://files.monsgeek.com ? This is the official download for the program, see: https://en.akkogear.com/download/ - despite the fact, that a slightly newer version has been published.

Our bot is unsophisticated and requires an actual human to verify, apologies.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image
This error shows up even with the dependency. It might need the `x86` version of `VCRedist` - we'd have to test to know for sure.
image

**Pull request merged**